Create markdown pages for docs site (#43)
onflow/flow-js-testing: docs/flow-token.md

---
title: FLOW Token Management
sidebar_title: FLOW Token
description: How to mint FLOW Token
---

> Some actions on the network will require account to have certain amount of FLOW token - transaction and storage fees,
> account creation, etc.
> 
> Framework provides a method to query balance with `getFlowBalance` and mint new tokens via `mintFlow`. You can find
> information how to use them below.

Reviewer: ```suggestion
Some actions on the network will require an account to have a certain amount of FLOW (tokens) - transaction and storage fees, account creation, etc.

Framework provides a method to query FLOW balances with `getFlowBalance` and mint new tokens via `mintFlow`.
```

### getFlowBalance(address)

> Returns current FlowToken balance of account specified by address

Reviewer: ```suggestion
Returns current FLOW token balance of the specified account.
```

- `address` - account address

Usage:

```javascript
import { init, emulator, getFlowBalance } from "flow-js-testing";

const main = async () => {
  const basePath = path.resolve(__dirname, "../cadence");
  const port = 8080;

  init(basePath, port);
  await emulator.start(port, false);

  const Alice = await getAccountAddress("Alice");

  try {
    const result = await getFlowBalance(Alice);
    console.log({ result });
  } catch (e) {
    console.log(e);
  }

  await emulator.stop();
};

main();
```

### mintFlow(recipient, amount)

> Sends transaction to mint specified amount of FLOW token and send it to recipient.

Reviewer: ```suggestion
Sends transaction to mint the specified amount of FLOW and send it to recipient.
```
